resources/.eslintrc-standards--test-override.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
I hadn't planned on it.  I just hate seeing red in my editor for things that really shouldn't be red.

"Max-depth exceeded".  Yeah, I did 2 describe blocks. DEAL WITH IT ESLINT!  xD